package.json

```diff
@@ -52,9 +52,8 @@
     "lint-staged": "lint-staged",
     "prepare": "husky install",
     "i18n:extract": "lingui extract",
-    "i18n:compile": "lingui compile",
-    "postinstall": "yarn i18n:extract && yarn i18n:compile",
-    "analyze": "yarn build && source-map-explorer 'build/static/js/*.js'"
```

Comment:
was this intended to get removed?

Reply:
I was intending to alter the compile and postinstall scripts but accidentally removed the analyze when resolving a merge, thanks for the pickup!